Homework - 02 Basic Python.ipynb

```diff
@@ -65,11 +201,42 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 13,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "What is your book's title?Alice in wonderland\n",
+      "This book is successfully checked out.\n",
+      "What is your name?yyy\n",
+      "{'Alice in wonderland': 'yyy', \"Hitchhiker's guide to the galaxy\": 'AA', 'Romeo and Juliet': False, 'WorldMap': 'CC'}\n"
+     ]
+    }
+   ],
    "source": [
-    "# your code"
+    "# your code\n",
+    "dict1 = { \"Alice in wonderland\" : False, \"Hitchhiker's guide to the galaxy\": \"AA\", \"Romeo and Juliet\": False,\"WorldMap\":\"CC\"}\n",
+    "a = input(\"What is your book's title?\")\n",
+    "def borrow(dict1,a):\n",
+    "\n",
+    "    if a not in dict1:\n",
+    "        print(\"This book is not in the library.\")\n",
+    "    elif dict1[a] != False:\n",
+    "        print(\"Sorry, this book has been checked out.\")\n",
+    "        y = input(\"Press 1 to see all the books that a particular user has checked out; Press 2 to see the user who has checked out a book\")\n",
+    "        if y == str(1):\n",
+    "            print([k for k,v in dict1.items() if v != False])\n",
+    "        if y == str(2):\n",
+    "            print([v for k,v in dict1.items() if k == a])\n",
```

Comment:
do you know what these two print statements are doing? they do not show all the books a particular user has checked out or show the user who has checked out a book.